.github/workflows/release.yaml

```diff
@@ -17,14 +20,52 @@ jobs:
           cache: 'npm'
       - run: npm ci
       - run: npm test
+
+  release:
+    needs: [test]
+    name: Release
+    runs-on: ubuntu-latest
+    outputs:
+      published: ${{ steps.changesets.outputs.published }}
+      publishedPackages: ${{ steps.changesets.outputs.publishedPackages }}
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
+          cache: 'npm'
+
+      - name: Install Dependencies
+        run: npm ci
+
+      - name: Create Release Pull Request or Publish to npm
+        id: changesets
+        uses: RomanHotsiy/changesets-action@v1
+        with:
+          # This expects you to have a script called release which does a build for your packages and calls changeset publish
+          publish: npm run release
+          commit: 'chore: 🔖 release new versions'
```

Comment:
Would be great to see release version in commit message

Reply:
Is changeset able to set git tags ?

Reply:
I believe changesets automatically adds tags during publishing.